titanium-rdf-api-java/src/main/java/eu/neverblink/jelly/convert/titanium/internal/TitaniumEncoderConverter.java

```diff
@@ -0,0 +1,66 @@
+package eu.neverblink.jelly.convert.titanium.internal;
+
+import eu.neverblink.jelly.core.InternalApi;
+import eu.neverblink.jelly.core.NodeEncoder;
+import eu.neverblink.jelly.core.ProtoEncoderConverter;
+import eu.neverblink.jelly.core.RdfProtoSerializationError;
+
+/**
+ * Converter for translating between Titanium RDF API nodes/terms and Jelly proto objects.
+ * <p>
+ * Quad class is used here, but is not intended to be used with the encoder.
+ * The only reason it's here is to satisfy the type signature of the trait.
```

Comment:
Quad class is no longer required by this trait. Please remove the second paragraph from this javadoc.